pg-snapshot.ver

```diff
@@ -1 +1 @@
-1-56
\ No newline at end of file
+1-56
```

Comment:
Let's use the most recent image `1-78`
https://catalog.redhat.com/software/containers/rhel8/postgresql-13/5ffdbdef73a65398111b8362

Reply:
Updated to the most recent image.